tools/ci_build/github/android/build_aar_package.py

```diff
@@ -151,7 +159,7 @@ def parse_args():
     parser.add_argument("--android_ndk_path", type=str, default=os.environ.get("ANDROID_NDK_HOME", ""),
                         help="Path to the Android NDK")
 
-    parser.add_argument('--build_dir', type=pathlib.Path, default=os.path.join(REPO_DIR, 'build_android_aar'),
+    parser.add_argument('--build_dir', type=pathlib.Path, default=os.path.join(REPO_DIR, 'build/android_aar'),
```

Comment:
>/ [](start = 95, length = 1)

nit: if we already use os.path.join(), how about `os.path.join(REPO_DIR, 'build', 'android_aar`)`?